### PR TITLE
new tutorial

### PR DESCRIPTION
**Pull request by kevinslin**
Revamping Dendron onboarding tutorial

### CODE REVIEW

File: vault/dendron.topic.multi-vault.md [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
 desc: ''
-updated: 1619551021163
+updated: 1620592648173
 created: 1605630383515
 ---
+
 ## Summary
 
-Multi vault lets you create multiple vaults for a given workspace. Each vault is a folder that can contain any number of hierarchies. 
+Multi vault refers to managing multiple [[vaults|dendron.concepts#vaults]] within your workspace
```
Comment (kpathakota) [inline vault/dendron.topic.multi-vault.md:11]: <3 this change. Much clearer. 

Nit: Missing period.

**Review by kpathakota — APPROVED** [review]
Will run through this with Nritya later today to see if there's anything else we catch but looks good to me.

File: vault/dendron.tutorial.md [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
 created: 1608051264282
 nav_order: 1.1
 ---
-- NOTE: if you'd rather follow a video tutorial, fellow dendrologist [Ian Jones](https://github.com/theianjones) made a fantastic video series which you can follow [here](https://egghead.io/courses/build-a-personal-knowledge-management-system-with-dendron-b24b)
+
 
 ## Pre-requisites
 
-This walkthrough will help you get started with Dendron. Before you start, make sure you have Dendron installed. If not, you can do so by following the instructions [[here|dendron.guides.install]].
+This walkthrough will help you get started with Dendron. Before you start, make sure you have Dendron installed and a workspace initialized.
```
Comment (hikchoi) [inline vault/dendron.tutorial.md:13]: I think this should be emphasized even more. I'd go as far as to say it should have a colored callout with an emoji in front of the text 😄  maybe bold / italic on  `workspace initialized` for now? 
```suggestion
This walkthrough will help you get started with Dendron. Before you start, make sure you have Dendron installed and **_a workspace initialized_**. 
```

File: vault/dendron.tutorial.recap.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+title: Recap
+desc: ''
+updated: 1620596685776
+created: 1595170096361
+nav_order: 1
+---
+
+## Summary
+
+This is a recap of the topics we covered in this tutorial and a bunch of additional topics we haven't.
```
Comment (hikchoi) [inline vault/dendron.tutorial.recap.md:12]: > and a bunch of additional topics we haven't.

Maybe this part should be a separate note or this note's title should change?

**Review by hikchoi — APPROVED** [review]
Love the recap. Nothing stuck out that looks like it needs changes but I left some comments for minor stuff.

**Pull request merged**